### DISCUSSION TITLE
service type LoadBalancer and Virtual IP

### DISCUSSION THREAD

**Discussion by miphip** [siderolabs/talos#5100] [Q&A] [answered]
If I run talos on my own machines, how can I set up an load balancer to support services of type LoadBalancer?
Can I use MetalLB?
MetalLB can use Virtual IP addresses similar to how talos does. Can they coexist?

**Comment by sergelogvinov** [comment, marked as answer]
MetalLB works well with Talos.

Virtual IP uses by contolplane only...
